src/utils/formHelperFunctions.ts

```diff
@@ -379,8 +379,6 @@ function textFieldShouldSubmit(servar: any, value: any) {
       methods = servar.metadata.login_methods;
       onlyPhone = methods.length === 1 && methods[0] === 'phone';
       return onlyPhone && value.length === 10;
-    case 'phone_number':
-      return value.length === 10;
```

Comment:
autosubmit gets triggered once the field gets completely filled out, but since phone numbers can have vairable lengths now, i'm not sure we can make any assumptions about completion

Reply:
Ok, if autosubmit is not supported for phone, then we should remove it from the dashboard.

Reply:
removed